apps/src/code-studio/pd/application_dashboard/detail_view_application_specific_questions.jsx

```diff
@@ -46,7 +46,7 @@ export default class DetailViewApplicationSpecificQuestions extends React.Compon
       principalApproval: this.props.initialPrincipalApproval
     };
 
-    this.sectionHeaders = props.applicationType === TEACHER ? _.omit(TeacherSectionHeaders, ['section5Submission']) : FacilitatorSectionHeaders;
+    this.sectionHeaders = props.applicationType === TEACHER ? _.omit(TeacherSectionHeaders, ['section5Submission', 'section6Submission']) : FacilitatorSectionHeaders;
```

Comment:
I don't think we need section5Submission in this list anymore - it doesn't exist anywhere else (I think they were renumbered)

Reply:
We do need this for rendering old applications